### ISSUE TITLE
Create a list of representative NIFS proposalIDs

### ISSUE THREAD

**Issue by ijiraq** [ijiraq/gemini_processing#10]
We need to have a set of proposalIDs that span some of the scope for the kinds of NIFS data that we might be processing.   Examples might be:
- Long exposures
- bright targets
- large numbers of targets.

Or, we could just randomly sample 10% of the programs to get a representative test set.

**Comment by Nat1405** [comment]
As JJ said there are 1742 NIFS proposals/programs (at least that I see in CADC). [Here is a list](https://www.dropbox.com/s/xnj1szx51hhr1db/proposals.txt?dl=0) of all of them; I generated it with the query "SELECT DISTINCT proposal_id FROM caom2.Observation AS o JOIN caom2.Plane AS p ON o.obsID=p.obsID WHERE instrument_name='NIFS'".

**Comment by dbohlender** [comment]
To restrict this to science programs I'd recommend restricting the search to program ID's like GN-2%.  That leaves 345 science programs, more in the ballpark I expected.  The "CAL" programs are facility calibration files, some of which will be used in the processing no doubt.

**Comment by Nat1405** [comment]
Just an update, I'm working a spreadsheet to make proposal selection easier:
[science_proposals.xls.zip](https://github.com/ijiraq/gemini_processing/files/4596470/science_proposals.xls.zip)

**Comment by Nat1405** [comment]
[Here's an updated spreadsheet with a proposed sample selection.](https://github.com/ijiraq/gemini_processing/files/4611025/science_proposals.xlsx.zip) See the "random sample" sheet for the selection. 

I've included some statistics below. I made this sample by random selection with some hand pruning at the end. I've included the average exposure time for 'OBJECT' frames (keep in mind this includes standard star observations so may be skewed) as a surrogate for brightness, the number of target names (again including standard star observations) and links to abstracts.
![image](https://user-images.githubusercontent.com/17349431/81590089-233bdc80-936f-11ea-8889-eada4df639ca.png)

Let me know if you want me to tweak the distributions more or break things down by program type (galactic, planetary, stellar). I wasn't sure how in-depth we wanted to go for this.
